lxc/network_forward.go

```diff
@@ -308,7 +334,11 @@ func (c *cmdNetworkForwardCreate) Run(cmd *cobra.Command, args []string) error {
 	}
 
 	if !c.global.flagQuiet {
-		fmt.Printf(i18n.G("Network forward %s created")+"\n", forward.ListenAddress)
+		if c.flagAllocate != "" {
+			fmt.Printf(i18n.G("Network forward created") + "\n")
+		} else {
+			fmt.Printf(i18n.G("Network forward %s created")+"\n", forward.ListenAddress)
```

Comment:
Lets put the listen address at the end to make it easier to parse, i.e `Network forward created: %s` and we can do this for all invocations.

Reply:
We might need to make a change to the client code to do this as the listen address is only returned in the `Location` header of the response, it's not returned by the client currently.